unitTests/magicbox/SuggestionsManagerTest.ts

```diff
@@ -404,17 +406,6 @@ export function SuggestionsManagerTest() {
             };
           }
 
-          let previewsBySuggestion: ISearchResultPreview[][];
-          const textPreviews = [['4K TV', 'Set of 4 pairs of socks'], ['The right spoon', '🔶', 'Right angle SATA cable']];
-          function buildPreviews() {
-            previewsBySuggestion = textPreviews.map(previewTextsForSuggestion => previewTextsForSuggestion.map(buildPreview));
-          }
-
-          const previewsPromisesWaitTimes = [1, 2];
-          function createPreviewsPromise(suggestionId: number) {
-            return Utils.resolveAfter(previewsPromisesWaitTimes[suggestionId], previewsBySuggestion[suggestionId]);
-          }
```

Comment:
This, and several other pieces of code, wasn't removed. I moved every tests in `describe('with previews')` into a new describe block under it named `describe('within timeout delay')`. I moved some functions and beforeEach calls out of `within timeout delay` into `with previews` so they could work with the new `describe('timing out')`.

tl;dr:
Went from
- **with previews**
  - ...functions
  - `beforeEach`
  - `afterEach`
  - ...tests
To
- **with previews**
  - ...most old functions
  - `beforeEach`
  - `afterEach`
  - **timing out**
    - ...new functions
    - `beforeEach`
    - ...new tests
  - **within timeout delay**
    - ...a few old functions
    - `beforeEach`
    - ...old tests